skins/oasis/modules/templates/Oasis_Index.php

```diff
@@ -95,9 +95,7 @@
 	<!-- Combined JS files and head scripts -->
 	<?= $jsFiles ?>
 <? endif ?>
-<? if ( isset($_COOKIE['tracking-opt-in-status']) ): ?>
-	<?= F::app()->renderView( 'CookieSyncerService', 'Index' ) ?>
-<? endif ?>
+<?= F::app()->renderView( 'CookieSyncerService', 'Index' ) ?>
```

Comment:
I believe that only essential scripts should be inlined in HTML. How about using AssetsManager to load this script asynchronously?
https://github.com/Wikia/app/blob/dev/extensions/wikia/AssetsManager/config.php

Reply:
@rogatty fix in 44a754fb8d8